src/extract/youtube.py

```
@@ -0,0 +1,142 @@
+    stats_response = (
+        youtube.videos()
+        .list(part="statistics,snippet", id=",".join(video_ids))
+        .execute()
+    )
+
+    # 시간 변환 함수 (UTC → timestamp)
+    def convert_to_timestamp(date_str):
+        try:
+            dt = datetime.strptime(date_str, "%Y-%m-%dT%H:%M:%SZ")
+            return int(time.mktime(dt.timetuple()))
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)